app/models/package_manager/base.rb

```diff
@@ -146,6 +147,16 @@ def self.add_version(dbproject, version_hash)
       existing.save
     end
 
+    def self.deprecate_versions(dbproject, version_hash)
+      case dbproject.platform.downcase
+      when "rubygems" # yanked gems will be omitted from project JSON versions
```

Comment:
:pray:

Reply:
I'm assuming these will get stashed in their respective package manager (at some point) yea?

Reply:
I kept it in here so it's next to the `add_version` method, but it would def make sense to extract to each class later too 👍

Reply:
I think that's generally been the pattern. Along with a constant check 🤷‍♂️